### ISSUE TITLE
support java?

### ISSUE THREAD

**Issue by Jie-Yuan** [stripe/agate#77]
support java?

**Comment by johnynek** [comment]
You can call scala code from Java. There is some knowledge involved but your IDE can probably help you.

This isn’t a big priority so far to make it easier but we would accept PRs that improve the Java accessibility if they were minimal, tested, documented, etc...

**Comment by non** [comment]
Yeah I agree with @johnynek -- if there are particular Java accessibility features that are needed a PR would help!
